SlicerCART/src/SlicerCART.py

```diff
@@ -94,16 +94,19 @@ def check_and_install_python_packages():
 KEYBOARD_SHORTCUTS_CONFIG_FILENAME = "keyboard_shortcuts_config.yml"
 CLASSIFICATION_CONFIG_FILENAME = "classification_config.yml"
 GENERAL_CONFIG_FILENAME = "general_config.yml"
+SEGMENTATION_CONFIG_FILENAME = "segmentation_config.yml"
```

Comment:
WHy not removing GENERAL_CONFIG_FILENAME = "general_config.yml" to keep only one config file?

Reply:
will have a follow-up PR for this issue